package.json

```diff
@@ -61,7 +61,7 @@
     "eslint-config-algolia": "20.0.0",
     "eslint-config-prettier": "8.5.0",
     "eslint-import-resolver-alias": "1.1.2",
-    "eslint-import-resolver-typescript": "2.7.1",
+    "eslint-import-resolver-typescript": "3.5.2",
```

Comment:
Updating this dep to use `project` config.

Reply:
I did look at the `CHANGELOG.md` for the package, and our configuration didn't use any of the depreciated features.